analyses/explore-spotify-variation.Rmd

```diff
@@ -14,6 +14,7 @@ Data was obtained from the [`R4DS Tidy Tuesday` repository](https://github.com/r
 # Load libraries
 suppressPackageStartupMessages({
   library(ggplot2)
+  library(patchwork)
 })
```

Comment:
Towards the top of the notebook during setup, can you set a seed to ensure UMAP reproducibility? Thanks! This is not tested, but should work!

```suggestion
set.seed(10)
})
```

Reply:
```suggestion
})

set.seed(10)
```